src/patternfly/components/Menu/menu.scss

```diff
@@ -1,252 +1,205 @@
-// @debug $menu; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
+:where(:root),
+:where(.#{$menu}) {
+  // * Menu
+  --#{$menu}--RowGap: var(--pf-t--global--spacer--sm);
+  --#{$menu}--Width: auto;
+  --#{$menu}--MinWidth: auto;
+  --#{$menu}--PaddingBlock: var(--pf-t--global--spacer--sm);
+  --#{$menu}--BackgroundColor: var(--pf-t--global--background--color--floating--default);
+  --#{$menu}--BoxShadow: var(--pf-t--global--box-shadow--md);
+  --#{$menu}--BorderRadius: var(--pf-t--global--border--radius--small);
+  --#{$menu}--OutlineOffset: calc(var(--pf-t--global--border--width--control--default) * -3);
+  --#{$menu}--ZIndex: var(--pf-t--global--Zindex--sm);
+  --#{$menu}--disabled--Color: var(--pf-t--global--text--color--disabled);
```

Comment:
It seems like `--#{$menu}--disabled--Color` is specific to plain button styling - can we update the var name to be more specific?